sci-biology/imagej/imagej-1.53t.ebuild

```diff
@@ -21,7 +21,7 @@ SRC_URI="https://imagej.nih.gov/ij/download/src/${MY_PN}${MY_PV}-src.zip
 LICENSE="public-domain" # http://imagej.net/disclaimer.html
 SLOT="0"
 
-KEYWORDS="~amd64"
+KEYWORDS="amd64"
```

Comment:
Since this is an overlay, we should as a matter of policy stick to unstable keyword always.

Reply:
Question 1: I tried KEYWORDS="~amd64" on both 1.53t and 1.54d, but `pkgcheck scan` complains of this, saying something along the lines of _overshadowing packages_. 
Is this of importance? or can I make them both ~amd64 and accept pkgcheck complaining?

Reply:
`
sci-biology/imagej
  RedundantVersion: version 1.53t: slot(0) keywords are overshadowed by version: 1.54d
`

Reply:
This is just warning you that the old version is a candidate for removal soon. You can ignore it for now.

Reply:
ignored. New commit has both on ~amd64 now